scripts/k8s_deploy_rapids_dask.sh

```diff
@@ -0,0 +1,125 @@
+#!/bin/bash
+# In order for this deployment to properly work across your cluster you will need to build the customer Docker image and push it out to your local Docker repository
+
+# This script will create a dask cluster including 1 Jupyter container,  1 Dask  scheduler, and N Dask workers.
+# The Dask workers can be scaled from  the K8S command line using `kubectl scale` commands or through jupyter using dask_kubernetes commands
+# See the included sample notebooks for details.
+
+NAMESPACE="rapids" # TODO: Pass through different ns end to end
+if [ -z "${NAMESPACE}" ]; then
+    NAMESPACE="rapids"
+fi
+
+
+function build_image() {
```

Comment:
The way this script is currently implemented it will build the Docker image locally, for multi-node systems we might need to add some scripting to push it to the local docker repository.

Within the next few months required library changes should be pushed upstream to the official RAPIDS or NGC docker images and we can remove this step altogether.